src/core/Form/MultiSelect/MultiSelect/MultiSelect.tsx

```diff
@@ -84,11 +84,11 @@ export interface MultiSelectProps<T extends MultiSelectData> {
   /** Text for screen reader to indicate how many items are selected */
   ariaSelectedAmountText: string;
   /** Text for screen reader indicating the amount of available options after filtering by typing. Will be read after the amount.
-   * E.g 'options available' as prop value would result in '<amount> options available' being read to screen reader upon removal.
+   * E.g 'options available' as prop value would result in '{amount} options available' being read to screen reader upon removal.
```

Comment:
Another nitpicking comment, but the ending should probably be "-- being read **by** screen reader upon removal" instead of "to" since it's the screen reader that is doing the reading. Same goes for the other instances of this doc format.

Reply:
This fix is now committed.